### ISSUE TITLE
[Feature request] Multiple docker instances

### ISSUE THREAD

**Issue by MartinStej** [gandrys/yocto-tools#3]
* Sometimes would be nice to have option to run multiple containers with Yocto
* From mount point of view, it is already done. Docker volume mapping solves problem with same Yocto directory.
 * Probably need to modify `yocto` command
 * Mounting/unmounting should not need to modify

**Comment by gandrys** [comment]
Thats true, it would be nice, but in the current stateless architecture and straightforward loop identification i would  have to rewrite entire "yocto" loop dev and container mapping. It is easy to do, I dont have time now. As workaround it is possible to install yocto-tools to other user account in linux and run second yocto env there. Usually only one running/working yocto env is enough. Moreover build of second yocto env would freez entire machine. Nerveless, I may add this feature in next releases. Feature plan is it rewrite it yocto-tools to python or to golang.  This tools was just proof of concept nothing else ....
